### PR TITLE
[12.0] add base_repair

### PR DESCRIPTION
**Pull request by marcelsavegnago**
This module extends the functionality of Odoo Repair module to add some basic features that are standard in other Odoo apps:

* Includes Repair Date field and add to the form
* Includes Repair Duration field and adds to the form
* Includes Responsible User field and add to the form
* Includes repair order search by Responsible User
* Includes grouping by Responsible User
* Includes grouping by Repair Date
* Implements Calendar view
* Implements PivotTable view

It does not include the timeline view in this module for fear of conflicting with the Odoo EE timeline component (I only work with the CE edition). If it's not a problem I can take the PR #524 and include it in this module

**Notes.**

1. In the repair_calendar_view module, fields are implemented that, honestly, are needed regardless of the view. And if I create other modules that implement other views, such as a timeline or pivot table, each of them will replace the views available in repair.action_repair_order_tree. 

2. In theory, this module eliminates the need for the repair_calendar_view module and I see no problem if the authors of the repair_calendar_view module want to be one of the authors of the base_repair module and included as contributors. Honestly, I think it's the right thing.

cc @rvalyi @renatonlima @mileo

### CODE REVIEW

File: base_repair/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,10 @@
+This module extends the functionality of Odoo Repair module to add some basic features that are standard in other Odoo apps:
+
+* Includes Repair Date field and add to the form
+* Includes Repato Duration field and adds to the form
```
Comment (rvalyi) [inline base_repair/readme/DESCRIPTION.rst]:
```suggestion
* Includes Repair Duration field and adds it to the form
```

File: base_repair/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1,10 @@
+This module extends the functionality of Odoo Repair module to add some basic features that are standard in other Odoo apps:
+
+* Includes Repair Date field and add to the form
```
Comment (rvalyi) [inline base_repair/readme/DESCRIPTION.rst]:
```suggestion
* Includes Repair Date field and adds it to the form
```

File: base_repair/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+* Includes Repato Duration field and adds to the form
+* Includes Responsible User field and add to the form
```
Comment (rvalyi) [inline base_repair/readme/DESCRIPTION.rst]:
```suggestion
* Includes Responsible User field and adds it to the form
```

**Comment by marcelsavegnago** [comment]
ping @OCA/manufacturing-maintainers

**Comment by mileo** [comment]
cc: @gabrielcardoso21

**Comment by marcelsavegnago** [comment]
ping @eduaparicio

**Review by eduaparicio — APPROVED** [review]

**Review by rvalyi — APPROVED** [review]
I approve. Also I chechecked the v13 and v14 Openupgrade repair  analysis logs to see if new fields would eventually be added an collide with these new ones and this is not the case (no such fields are added).

**Pull request merged**